gap/examples.gi

```diff
@@ -658,6 +1081,307 @@ function(filt, m, n)
   return D;
 end);
 
+InstallMethod(KneserGraphCons,
+"for IsImmutableDigraph, integer, integer",
+[IsImmutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D;
+  D := MakeImmutable(KneserGraphCons(IsMutableDigraph, n, k));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  SetIsRegularDigraph(D, true);
+  SetIsVertexTransitive(D, true);
+  SetIsEdgeTransitive(D, true);
+  if n >= 2 * k then
+    SetChromaticNumber(D, n - 2 * k + 2);
+  else
+    SetChromaticNumber(D, 1);
```

Comment:
Could you please add a test for this line? Also, the only digraphs with chromatic number 1 are empty, so I presume you could/should also set that property too, as you did for e.g. `KellerGraphCons`?
```suggestion
    SetChromaticNumber(D, 1);
    SetIsEmptyDigraph(D, true);
```